plugins/src/plugins/mps/cap_station.cpp

```diff
@@ -278,3 +281,75 @@ bool CapStation::pose_in_shelf_right(const gzwrap::Pose3d &puck_pose)
   return pose_hit(puck_pose, shelf_right_pose(),0.05);
 }
 
+void CapStation::decide_broken_state()
```

Comment:
Please add documentation for the added functions. Even if the existing code is not well-documented, it's always good to start with better practices.